motr/idx_dix.c

```diff
@@ -459,7 +459,7 @@ static void cas_put_ast(struct m0_sm_group *grp, struct m0_sm_ast *ast)
 	struct m0_op_idx        *oi = dix_req->idr_oi;
 	struct m0_cas_id         idx;
 	struct m0_cas_req       *creq = &dix_req->idr_creq;
-	uint32_t                 flags = 0;
+	uint32_t                 flags = COF_VERSIONED|COF_OVERWRITE;
```

Comment:
And when COF_VERSIONED is set, the COF_OVERWRITE must be set?

Reply:
Correct, though note that this PR doesn't cover NEXT. From the CAS header docs, COF_OVERWRITE is required to enable COF_VERSIONED. Yeah, it seems like there will be no way to avoid COF_OVERWRITE

Reply:
Thank you.

Reply:
OK.